doc/reference/tooling/tt_cli/start.rst

```diff
@@ -1,54 +1,62 @@
 .. _tt-start:
 
-Starting a Tarantool instance
-=============================
+Starting Tarantool applications
+===============================
 
 ..  code-block:: console
 
-    $ tt start {INSTANCE | APPLICATION[:APP_INSTANCE]}
+    $ tt start {APPLICATION[:APP_INSTANCE] | SINGLE_INSTANCE}
 
-``tt start`` starts Tarantool applications.
+``tt start`` starts Tarantool applications. The application files must be stored
+inside the ``instances_enabled`` directory specified in the :ref:`tt configuration file <tt-config_file_app>`.
+For detailed instructions on preparing and running Tarantool applications, see
+:ref:`admin-instance-environment-overview` and :ref:`admin-start_stop_instance`.
 
-Details
--------
+When called without arguments, starts all enabled applications in the current environment.
 
-The application files must be stored inside the ``instances_enabled``
-directory specified in the :ref:`tt configuration file <tt-config_file_app>`.
-The ``APPLICATION`` value can be:
+Cluster application
```

Comment:
As we discussed, let's consider restructuring of such pages to reflect that multi-instance apps can both use new and old config.